src/sys/gtl.c

```diff
@@ -51,10 +51,12 @@ typedef struct FnBundle {
 s32 gtlD_80040CF0 = 0;
 u32 gtlFrameCounter = 0;
 s32 gtlDrawnFrameCounter = 0;
-UcodeInfo gtlD_80040CFC[] = { { NULL, NULL }, { NULL, NULL }, { gspF3DEX2_NoN_fifoTextStart, gspF3DEX2_NoN_fifoDataStart },
-                              { NULL, NULL }, { NULL, NULL }, { NULL, NULL },
-                              { NULL, NULL }, { NULL, NULL }, { gspL3DEX2_fifoTextStart, gspL3DEX2_fifoDataStart },
-                              { NULL, NULL }, { NULL, NULL } };
+UcodeInfo gtlD_80040CFC[] = {
+    { NULL, NULL }, { NULL, NULL }, { gspF3DEX2_NoN_fifoTextStart, gspF3DEX2_NoN_fifoDataStart },
```

Comment:
This is how I'd like to see long function arguments being handled as well. All arguments in a block 4 spaces deeper, but with some manual control over the linebreaks, in order to group 'related' properties manually (e.g. x/y).